onboard/catkin_ws/src/cv/scripts/test_images.py

```diff
@@ -69,10 +69,9 @@ def run_still(self):
         """Publish a still image to topic with the specified framerate."""
 
         image = cv2.imread(self.feed_path, cv2.IMREAD_COLOR)
-        image_msg = self.cv_bridge.cv2_to_imgmsg(image, 'bgr8')
         loop_rate = rospy.Rate(self.framerate)
-
         while not rospy.is_shutdown():
+            image_msg = self.image_tools.convert_to_ros_compressed_msg(image)
             self.image_publisher.publish(image_msg)
```

Comment:
Why is the conversion being done in the loop now? Is there a possibility for `image` to change?

Reply:
Put conversion outside loop.